TipCatDotNet.Api/Services/HospitalityFacilities/PaymentService.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading;
+using System.Threading.Tasks;
+using CSharpFunctionalExtensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using TipCatDotNet.Api.Data;
+using TipCatDotNet.Api.Models.HospitalityFacilities;
+using TipCatDotNet.Api.Data.Models.HospitalityFacility;
+
+
+namespace TipCatDotNet.Api.Services.HospitalityFacilities
+{
+    public class PaymentService : IPaymentService
+    {
+        public PaymentService(ILoggerFactory loggerFactory, AetherDbContext context)
+        {
+            _context = context;
+            _logger = loggerFactory.CreateLogger<MemberService>();
+        }
+
+
+        public Task<Result<PaymentDetailsResponse>> Pay(PaymentRequest paymentRequest, CancellationToken cancellationToken = default)
+        {
+            return Result.Success()
+                .EnsureMemberExists(_context, paymentRequest.MemberId, cancellationToken)
```

Comment:
The domain logic falls over different classes. Probably, we should stop make extensions this way in the future